esti/hooks_test.go

```diff
@@ -29,18 +46,33 @@ func TestHooksSuccess(t *testing.T) {
 	require.NoError(t, err, "failed to commit initial content")
 	require.Equal(t, http.StatusCreated, commitResp.StatusCode())
 
-	_, err = responseWithTimeout(server, 1*time.Minute) // pre-commit action triggered on action upload, flush buffer
+	webhookData, err := responseWithTimeout(server, 1*time.Minute) // pre-commit action triggered on action upload, flush buffer
 	require.NoError(t, err)
 
+	require.NoError(t, webhookData.err, "error on pre commit serving")
+	decoder := json.NewDecoder(bytes.NewReader(webhookData.data))
+	var preCommitEvent webhookEventInfo
+	require.NoError(t, decoder.Decode(&preCommitEvent), "reading pre-commit data")
+	appendRes(preCommitEvent)
+
 	t.Run("commit merge test", func(t *testing.T) {
 		testCommitMerge(t, ctx, repo)
+
 	})
 	t.Run("create delete branch test", func(t *testing.T) {
 		testCreateDeleteBranch(t, ctx, repo)
 	})
 	t.Run("create delete tag test", func(t *testing.T) {
 		testCreateDeleteTag(t, ctx, repo)
 	})
+
+	t.Log("check runs are sorted in descending order")
```

Comment:
Thanks for adding this check, we didn't realize how important and tricky this might be until this PR.